torch/testing/_internal/common_nn.py

```diff
@@ -3935,6 +3943,23 @@ def single_batch_reference_fn(input, parameters, module):
         skip_double=TEST_WITH_ROCM,
         pickle=False,
     ),
+    dict(
+        module_name='PairwiseDistance',
+        input_fn=lambda: (torch.randn(10, 8), torch.randn(10, 8)),
+    ),
```

Comment:
Was PairwiseDistance untested in these dicts before? If so, might be worth adding a case or two for broadcasting as well

Reply:
Done :)